test/e2e/user_management_test.go

```diff
@@ -170,15 +179,18 @@ func (s *userManagementTestSuite) TestUserDeactivation() {
 			RequireConditions(ConditionSet(Default(), ApprovedByAdmin())...).
 			Execute().Resources()
 
-		// Get the base tier that has deactivation disabled
-		baseDeactivationDisabledTier, err := hostAwait.WaitForNSTemplateTier("basedeactivationdisabled")
+		// TODO remove once UserTier migration is completed
+		s.promoteToDefaultUserTier(hostAwait.Client, murMember1)
```

Comment:
we don't need that when the user will be promoted to `nodeactivation` tier in the following lines, right?

Reply:
Removed this and other cases in a6cd3440370feefacd130d319af1cd0fc90c9990

Reply:
Right again! Removed it c6022abad03a905ac5e3691007be3e603a190b62